dependencies.gradle

```diff
@@ -1,11 +1,11 @@
-ext.composeVersion = "1.0.0-alpha07"
+ext.composeVersion = "1.0.0-alpha09"
 
 dependencyManagement {
     dependencies {
         dependency "androidx.appcompat:appcompat:1.3.0-alpha02"
         dependency "androidx.compose.material:material:$composeVersion"
         dependency "androidx.compose.ui:ui:$composeVersion"
-        dependency "androidx.ui:ui-tooling:$composeVersion"
+        dependency "androidx.ui:ui-tooling:1.0.0-alpha07"
```

Comment:
so I'm a little confused here:
The version >alpha7 aren't available
https://mvnrepository.com/artifact/androidx.ui/ui-tooling

the name seems to be changed to androidx.compse.ui:ui-tooling,
https://stackoverflow.com/questions/64961333/gradle-cant-resolve-androidx-uiui-tooling-dependency-by-google-repository

however this doesn't seem to contain the `@Preview` annotation ?

Reply:
Ah, the artifact is renamed. Use `dependency "androidx.compose.ui:ui-tooling:$composeVersion"`